### PR TITLE
feat: fix user onboarding states

### PR DESCRIPTION
**Pull request by abiudrn**

### CODE REVIEW

File: test/widgets_tests/redux/actions/onboarding/security_questions/get_security_questions_action_test.dart [hunk 1]
```diff
@@ -70,53 +69,52 @@ void main() {
...
-                  );
-                } catch (e) {
-                  err = e;
-                }
-              },
-            );
-          },
-        ),
-      );
+    // testWidgets('should dispatch action and catch error if is pin reset',
```
Comment (byronkimani) [inline test/widgets_tests/redux/actions/onboarding/security_questions/get_security_questions_action_test.dart:72]: unused code

Comment (johnngugi) [inline test/widgets_tests/redux/actions/onboarding/security_questions/get_security_questions_action_test.dart]: Dead code

**Review by johnngugi — CHANGES_REQUESTED** [review]

**Review by johnngugi — APPROVED** [review]

File: test/widgets_tests/redux/actions/onboarding/security_questions/get_security_questions_action_test.dart (hunk 1 above)
Comment (abiudrn) [reply test/widgets_tests/redux/actions/onboarding/security_questions/get_security_questions_action_test.dart]: Fixed

Comment (abiudrn) [reply test/widgets_tests/redux/actions/onboarding/security_questions/get_security_questions_action_test.dart]: Addressed

**Review by Paulmburu — APPROVED** [review]

**Review by EugeneDevv — APPROVED** [review]

**Review by byronkimani — APPROVED** [review]

**Pull request merged**